src/status_im/contexts/centralized_metrics/core.cljs

```diff
@@ -0,0 +1,88 @@
+(ns status-im.contexts.centralized-metrics.core
+  (:require
+    [legacy.status-im.utils.build :as build]
+    [native-module.core :as native-module]
+    [re-frame.interceptor :as interceptor]
+    [react-native.platform :as platform]
+    [taoensso.timbre :as log]
+    [utils.re-frame :as rf]))
+
+(defn user-journey-event
+  [action]
+  {:metric
+   {:eventName  "user-journey"
+    :platform   platform/os
+    :appVersion build/app-short-version
+    :eventValue {:action action}}})
+
+(def app-started-event "app-started")
+(def new-to-status-event "clicked-on-new-to-status")
+(def navigate-to-create-profile-event "navigate-to-create-profile")
+(def communities-tab-clicked "communities-tab-clicked")
+(def wallet-tab-clicked "wallet-tab-clicked")
+(def chats-tab-clicked "chats-tab-clicked")
+
+(defn track-view-id-event
+  [view-id]
+  (case view-id
+    :communities-stack (user-journey-event communities-tab-clicked)
+    :chats-stack       (user-journey-event chats-tab-clicked)
+    :wallet-stack      (user-journey-event wallet-tab-clicked)
+    nil))
+
+(defn tracked-events
```

Comment:
It would be good if we could separate the general abstractions from the concrete tracking. The abstractions are basically not changing, but the tracking code will. For example, you could put all tracking logic into `centralized_metrics/tracking.cljs` or `centralized_metrics/user_journeys.cljs`.